cachito/web/content_manifest.py

```diff
@@ -1,7 +1,12 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 from copy import deepcopy
+import pkg_resources
+import os
+import re
+import urllib.parse
 
 import flask
+from cachito.errors import ContentManifestError, ValidationError
```

Comment:
I'd like to keep stdlib, 3rd party and cachito imports separated by an empty line. I think `copy, os, re, urllib` would be the stdlib modules, `flask` and `pkg_resources` 3rd party.

Reply:
Done!